libnmstate/schema.py

```diff
@@ -290,3 +292,20 @@ class OVSBridge(object):
     PORT_NAME = 'name'
     PORT_VLAN_MODE = 'vlan-mode'
     PORT_ACCESS_TAG = 'access-tag'
+
+
+class Team(object):
```

Comment:
No need for `object`.

Reply:
If there is no need for object, why are we using it in the other schema classes? Thanks!

Reply:
Okay, this is a leftover from python2 support. I am removing it. Thanks!

Reply:
Just leftover of python2 stuff.